egs/multi_en/s5/run.sh

```diff
@@ -137,19 +140,22 @@ if [ $stage -le 7 ]; then
   mfccdir=mfcc
   corpora="hub4_en fisher librispeech_100 librispeech_360 librispeech_500 swbd tedlium wsj"
   for c in $corpora; do
-    data=data/$c/train
-    steps/make_mfcc.sh --mfcc-config conf/mfcc.conf \
-      --cmd "$train_cmd" --nj 40 \
-      $data exp/make_mfcc/$c/train || exit 1;
-    steps/compute_cmvn_stats.sh \
-      $data exp/make_mfcc/$c/train || exit 1;
+    (
+     data=data/$c/train
+     steps/make_mfcc.sh --mfcc-config conf/mfcc.conf \
+       --cmd "$train_cmd" --nj 40 \
+       $data exp/make_mfcc/$c/train || touch $data/.error
```

Comment:
Are you checking if there is an error in a later stage? Also you should first remove the `rm -f` the `.error` files.

Reply:
hmm I actually made it back to || exit 1; since I think we should quit if the feature extraction fails

Reply:
`|| exit 1` will have no effect inside a subshell running in background.

Reply:
fixed